### ISSUE TITLE
Repo is obsolete and should probably be deleted

### ISSUE THREAD

**Issue by hlapp** [phyloref/wop-2016-paper#1]
There doesn't seem to be any content in here, and the manuscript was never finished for WOP 2016. Instead, the manuscript was [picked up again, submitted to, and accepted for WOP 2017](http://daselab.cs.wright.edu/pub2/wop2017-trees.pdf), but that version was not tracked or archived here.

**Comment by gaurav** [comment]
Agreed! If there was any content here, we could have archived it in the website repo. As it stands, I think we should just delete this repository.
